common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -98,6 +98,7 @@ def _student_or_public_view(self, context, view):
         if view == STUDENT_VIEW:
             fragment_context.update({
                 'show_bookmark_button': child_context.get('show_bookmark_button', not is_child_of_vertical),
+                'is_microfrontend_embed': child_context.get('is_microfrontend_embed', False),
```

Comment:
FWIW, I don't think you need to be this specific. The VerticalBlock doesn't need to know that it's part of a microfrontend embed (and someone hacking at this layer may not even understand the significance of that). This part of the code just needs to know to not show the title. There might be a number of reasons why it should do that (maybe for mobile display, even, I dunno).

So I think just having `show_title` and defaulting it to `True` would work well here.

Reply:
Sounds good.